src/site/_includes/spaces.njk

```diff
@@ -0,0 +1,158 @@
+
+    <article class="card flow hidden">
+      <div class="card__header">
+        <div class="avatars">
+        {% for host in space.data.hosts %}
+          <img class="avatar" src="{{ host.image }}" alt="alt" decoding="async" height="40" loading="lazy" width="40">
+        {% endfor %}
+        </div>
+      </div>
+      <div class="card__content flow">
+        <h2 class="card__heading text-size-3">
+          {{ space.data.title }}
+        </h2>
+        <ul class="hosts">
+          {% for host in space.data.hosts %}
+            <li>
+                {{ host.name }}
+              {% if host.host %} <span class="pill" data-inactive>{{ 'i18n.spaces.host' | i18n(locale) }}</span> {% endif %}
+            </li>
+          {% endfor %}
+        </ul>
+        <time class="eyebrow">{{ space.data.event_date | prettyDate('DATE_FULL') }}</time>
+        <p class="gap-top-size-1 gap-bottom-size-2">{{ space.data.description }}</p>
+      </div>
+      <a class="button" href="{{ space.data.event_date | calendarLink(space.data.title, space.data.description) }}">
+        {% include "icons/calendar.svg" %}
+        <span>{{ 'i18n.spaces.add_to_calendar' | i18n(locale) }}</span>
+      </a>
+    </article>
+    {% endif %}
+  {% endfor %}
+
+  </div>
+</div>
+
+
+<div class="spaces spaces--past wrapper region all-center">
+  <h2 class="text-size-4  gap-bottom-size-2">{{ 'i18n.spaces.past_spaces' | i18n(locale) }}</h2>
+
+  {% if not pastSpaces.length %}
+  <p>A friendly message reminding that the recordings will appear here</p>
+  {% endif %}
+
+  <table>
+    {% for space in pastSpaces %}
+      <tr>
+        <td class="eyebrow all-center">
+          <p>
+            {{ space.data.event_date | prettyDate('') }}
+          </p>
+        </td>
+        <td class="pad-block-size-1 pad-inline-size-1">
+          <div class="avatars">
+            {% for host in space.data.hosts %}
+              <img class="avatar" src="{{ host.image }}" alt="alt" decoding="async" height="40" loading="lazy" width="40">
+            {% endfor %}
+          </div>
+          <p class="gap-top-size-1 eyebrow sm-only">
+            {{ space.data.event_date | prettyDate('DATE_SHORTER') }}
+          </p>
+        </td>
+        <td class="flow pad-block-size-1">
+          <h2 class="text-size-2">
+            {{ space.data.title }}
+          </h2>
+          <p>
+            {% for host in space.data.hosts %}
+              <span class="host">{{ host.name }} {% if host.host %} <span class="pill" data-inactive>{{ 'i18n.spaces.host' | i18n(locale) }}</span> {% endif %}</span>
+            {% endfor %}
+          <details class="ellipsis">
+            <summary><span class="ellipsis__full">{{ space.data.description }}</span><span class="ellipsis__more">{{ 'i18n.spaces.see_more' | i18n(locale) }}</span></summary>
+          </details>
+          <p><audio src="{{ space.data.audio }}" class="audio-player__element" controls="" id="audio-player"></audio></p>
```

Comment:
You might end up with multiple `audio` elements on the page with the same `id="audio-player"` attribute.